### PR TITLE
[DRAFT] Initial example notebook

### PR DESCRIPTION
**Pull request by shivdeep-singh-ibm**
## Why are these changes needed?
Added notebook to test transforms locally.

### CODE REVIEW

File: examples/Makefile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+REPOROOT=../
+# Use make help, to see the available rules
+
+include ../.make.defaults
+
+#venv::	.transforms.venv 	.defaults.lib-src-venv
+venv:: .defaults.lib-src-venv
```
Comment (daw3rd) [inline examples/Makefile:7]: When I `make venv` on my mac m1, I get the following, eventually.
```
$ make venv
...
Installing collected packages: huggingface-hub, tokenizers, transformers
  Attempting uninstall: huggingface-hub
    Found existing installation: huggingface-hub 0.23.0
    Uninstalling huggingface-hub-0.23.0:
      Successfully uninstalled huggingface-hub-0.23.0
  Attempting uninstall: tokenizers
    Found existing installation: tokenizers 0.15.2
    Uninstalling tokenizers-0.15.2:
      Successfully uninstalled tokenizers-0.15.2
  Attempting uninstall: transformers
    Found existing installation: transformers 4.38.2
    Uninstalling transformers-4.38.2:
      Successfully uninstalled transformers-4.38.2
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
datasets 2.17.1 requires huggingface-hub>=0.19.4, but you have huggingface-hub 0.17.3 which is incompatible.
Successfully installed huggingface-hub-0.17.3 tokenizers-0.14.1 transformers-4.35.0
$ 

```

File: examples/Makefile [hunk 2]
```diff
@@ -0,0 +1,15 @@
+REPOROOT=../
+# Use make help, to see the available rules
+
+include ../.make.defaults
+
+#venv::	.transforms.venv 	.defaults.lib-src-venv
```
Comment (daw3rd) [inline examples/Makefile:6]: You may want to add a `clean` target to remove the venv (I think you could use .defaults.clean)

File: examples/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+## Demo Notebook
+
+We can launch a jupyter notebook to try out the library or transforms or to build a usecase.
```
Comment (daw3rd) [inline examples/README.md:3]: We need to either a) tell the user where to get an input data set and where to put it or b) include one in the repo.  However, if (b), it should not be large - say not larger than a couple of MB if possible.

**Review by daw3rd — CHANGES_REQUESTED** [review]

File: examples/README.md (hunk 3 above)
Comment (shivdeep-singh-ibm) [reply examples/README.md:3]: Addressed in the rebased pull req on another branch https://github.com/IBM/data-prep-lab/pull/89

Updated Notebook with instructions to get data

**Comment by shivdeep-singh-ibm** [comment]
Rebased pull req on another branch https://github.com/IBM/data-prep-lab/pull/89

**Pull request closed without merging**